config/routes.rb

```diff
@@ -183,7 +183,10 @@
 
     resources :ownership_calls, only: :index
     resources :webauthn_credentials, only: :destroy
-    get 'webauthn_verification/:webauthn_token', to: 'webauthn_verifications#prompt', as: 'webauthn_verification'
+    resource :webauthn_verification, only: [] do
```

Comment:
Not dumb at all! To my knowledge, you define an only clause to keep the [specified actions for a resource](https://guides.rubyonrails.org/routing.html#specifying-a-controller-to-use) (eg. `update`, `show`, `edit`). Since this controller doesn't implement any of those default actions, I put it as `[]`. I put the "custom" routes in the resource clause to help with grouping and remove some repetition in the code.

Reply:
This might be an occasion for a EOL comment. Something like "# 'only' is empty because this controller has custom actions rather than 'show', 'create' etc."

Reply:
I'm leaning towards refraining from adding a comment. There's 4-5 occasions of this in the routes file already so this pattern is not going outside any unexpected code. If you feel strongly, I'll add it in!

Reply:
Gotcha. In that case we should leave it uncommented.